webui/server.py

```
@@ -0,0 +1,195 @@
+    )
+
+# Mount static files for js and other assets
+app.mount("/static", StaticFiles(directory=Path(__file__).parent), name="static")
+
+# Store WebSocket connections
+
+# Store active WebSocket connections
+active_connections: List[WebSocket] = []
+
+@app.websocket("/ws")
+async def websocket_endpoint(websocket: WebSocket):
```

Ruff (0.8.2)

`websocket_endpoint` is too complex (14 > 10)

(C901)